src/oge/consumed.py

```diff
@@ -438,13 +441,13 @@ def _load_rates(self):
                         axis=1,
                     )
 
-                # Cut off emissions at 9 hours after UTC year
-                emissions = emissions[: f"{self.year+1}-01-01 09:00:00+00:00"]
+                # Cut off emissions at 8 hours after UTC year
+                emissions = emissions[: f"{self.year+1}-01-01 08:00:00+00:00"]
```

Comment:
How does it work? I thought the emission would be hourly over a full year in UTC time. Are we doing this in Pacific Time instead. And why was it nine hours before, did we have Alaska in 2021 and not 2022?

Reply:
So this is just saying that we only want to do the calculations through midnight pacific time on 12/31, which would be 8am UTC time on Jan 1. Not sure why this was previously set to 9 hours, since we do not do the consumed calculations for AK or HI. I'm also not sure why this didn't raise a key error in previous years, but it does now, since 2023-01-01 9:00 does not exist in the data.

Reply:
Should we also set the lower bound? So, it starts on 2022-01-01 Pacific Time, i.e,  do:
```
emissions = emissions[f"{self.year}-01-01 08:00:00+00:00": f"{self.year+1}-01-01 08:00:00+00:00"]
```
Right now, we probably have the last 8 hours in 2021/12/31.